### DISCUSSION TITLE
Hide tab bar property

### DISCUSSION THREAD

**Discussion by juanfrangm** [caplin/FlexLayout#423] [Q&A]
Hi,

I'm currently working on a project that's using FlexLayout, and the user wants a specific panel to hide the tabs bar under certain scenarios.

![image](https://github.com/caplin/FlexLayout/assets/45510820/130d0398-e59e-48c3-97f7-7fbadc8f9413)

Let's say for example:
- When page is in edit mode -> show the tabs bar
- When page is in view mode -> hide the tabs bar

So we were wondering if there is a current property for this or if there are plans in the future to implement this feature.

Thanks.

**Comment by Einar-san** [comment]
Hey there, the property on the tab is called 'enableTabStrip,' and you can find more information about it in the readme :)
